modules/cluster-logging-collector-external.adoc

```diff
@@ -134,21 +186,33 @@ For example:
   </store>
 ----
 
-. Optionally, create a secret named `secure-forward` in the `openshift-logging` namespace.
+. Create a ConfigMap named `secure-forward` in the `openshift-logging` namespace from the configuration file:
++
+----
+$ oc create configmap secure-forward --from-file=secure-forward.conf -n openshift-logging
+----
+
+. Optionally, import any secrets required for the receiver:
 +
 ----
-$ oc create secret secure-forward --from-file=<arbitrary-name-of-key>=<file-name-of-ssl-private-key> --from-file=<arbitrary-name-of-cert>=<file-name-of-ssl-cert>
+$ oc create secret generic secure-forward --from-file=<arbitrary-name-of-key1=cert_file_from_fluentd_receiver --from-literal=shared_key=value_from_fluentd_receiver
```

Comment:
Is this missing a \> somewhere? The `oc` command doesn't make sense to me as written.